api/src/main/resources/rdr.yaml

```diff
@@ -0,0 +1,202 @@
+#This file provide the end points for interaction with RDR
+
+
+swagger: '2.0'
+
+info:
+  title: RDR
+  description: |
+    Service for RDR Interactions.
+  version: "0.1"
+#The host will get updated as per the environment it will be pmi-drc-api-test for dev and test
```

Comment:
Please put yaml comments in descriptions instead of using `#`. The latter aren't well-supported by tools, since they're not a real part of the object model for the doc.